wurst/closures/Execute.wurst

```diff
@@ -0,0 +1,32 @@
+package Execute
+
+public interface ForForceCallback
+	function callback()
+
+force dummy = CreateForce()
+
+init
+	ForceAddPlayer(dummy, GetLocalPlayer())
+
+function callback() returns boolean
```

Comment:
not a suitable function name for this internal fn

Reply:
What name would be more suitable?

Reply:
May I suggest `_currentCallback`? @Frotty ?

Reply:
why underscore in front? the other libs (closureforgroups) use `currentCallback` which should be fine

Reply:
That's fine too. Why underscore? Costs very little and benefit is slightly improved indication of "this is not public, and its use is very specific/controlled by this package"